### ISSUE TITLE
Furex eats my frames

### ISSUE THREAD

**Issue by hamogor** [yohamta/furex#24]
Currently my project gets about 500 fps with `ebiten.ActualFPS()`. After grabbing the basic example from the readme my frames drop to about 20. Is this a known issue? Project seems fantastic and I'd really like to work with it.

**Comment by yohamta** [comment]
Thanks for submitting the issue. I am not sure the cause, what if setting the FPSMode to VsyncOff?
```go
	ebiten.SetFPSMode(ebiten.FPSModeVsyncOffMaximum)
```
I am using furex in my project and the FPS is around 60 with `FPSModeVsyncOffMaximum`.
![image](https://user-images.githubusercontent.com/1475839/196319404-c2ce58e0-0b76-4f78-a3ae-51ec19958d8e.png)

FPS is rendered with the below code.
```go
	ebitenutil.DebugPrint(screen, fmt.Sprintf("FPS: %.2f", ebiten.CurrentFPS()))
```

**Comment by yohamta** [comment]
Sorry, maybe it was a bug in the example. I think it is already fixed in `v2.2.1`. Please reopen the issue if it still happens.
